src/identity/components/GlobalHeader/GlobalHeaderDropdown/index.tsx

```diff
@@ -112,7 +113,7 @@ export class GlobalHeaderDropdown extends React.Component<Props, State> {
     this.setState({showTypeAheadMenu: !showTypeAheadMenu})
   }
 
-  private renderMainMenuOptions = () => {
+  private renderMainMenuOptions = onCollapse => {
```

Comment:
Point of this is to pass in the `onCollapse` function that forces the dropdown to close when an option is clicked, per design instructions. This wasn't necessary before when a page reload was occurring -- which would necessarily reload the dropdown.

This is to occur _only_ when the main menu options are clicked, because the user needs to be able to access the switch-account menu without the dropdown closing. That's the reason we're not passing `onCollapse` into the entire dropdown.

Reply:
FYI @juliajames12